eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgAuditDb.scala

```diff
@@ -189,17 +204,17 @@ class PgAuditDb(implicit ds: DataSource) extends AuditDb with Logging {
         statement.setString(3, errorName)
         statement.setString(4, errorMessage)
         statement.setBoolean(5, e.isFatal)
-        statement.setLong(6, System.currentTimeMillis)
+        statement.setTimestamp(6, Timestamp.from(Instant.now()))
         statement.executeUpdate()
       }
     }
   }
 
   override def listSent(from: Long, to: Long): Seq[PaymentSent] =
     inTransaction { pg =>
-      using(pg.prepareStatement("SELECT * FROM sent WHERE timestamp >= ? AND timestamp < ?")) { statement =>
-        statement.setLong(1, from)
-        statement.setLong(2, to)
+      using(pg.prepareStatement("SELECT * FROM sent WHERE timestamp BETWEEN ? AND ? ORDER BY timestamp")) { statement =>
```

Comment:
NB: added an `ORDER BY TIMESTAMP` here.

Reply:
In fact we can't, because results are aggregated in a `Map`. I removed the `ORDER BY` instead in 6b20b25.